tests/integration/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: "3.7"
+services:
+  server:
+    build:
+      context: ../..
+      dockerfile: tests/integration/Dockerfile.integration-test.yml
+    ports:
+      - "6060:6060"
+    depends_on:
+      - postgres
+    environment:
+      - HIBP_KANON_API_TOKEN
+      - HIBP_API_TOKEN
+      - FXA_ENABLED=1
+      - LOGOS_ORIGIN=null
+      - BREACH_RESOLUTION_ENABLED=false
+      - MONITOR_FXA_PASSWORD
+  postgres:
+    image: postgres
+    restart: always
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_DB=blurts
+    expose:
+      - "5432"
+  migration:
+    image: blurts-server_server:latest
```

Comment:
Just following up on our Slack thread... Should we change this line to `integration_server:latest` ?